packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -60,7 +75,12 @@ export const initializeNodeEnvironment: EnvironmentInitializer<
     child.stdout?.setEncoding('utf8');
     child.stderr?.setEncoding('utf8');
     child.stdout?.on('data', console.log); // eslint-disable-line no-console
-    child.stderr?.on('data', console.error); // eslint-disable-line no-console
+
+    let lastSeenError: string | undefined = undefined;
+    child.stderr?.on('data', (chunk) => {
+        console.error(chunk); // eslint-disable-line no-console
+        lastSeenError = chunk;
```

Comment:
In the case of a process crash (e.g. out of memoty) the entire output might not fit in a single chunk. Can you please verify?

Maybe we could include the last second of stderr output instead?

Reply:
You were right, I changed the logic to collect all errors if the time difference between them is less than 5 seconds.